wolvic/wolvic_content_client.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2012 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Nit: I know this might be eventually part of Wolvic, but we either remove this or use the proper year.

Reply:
I'm not familiar with GitHub, but it doesn't seem to be showing it as an integrated change. This copyright has already been deleted. :/